website/docs/cairo-1/06-interacting-with-starknet/06-multicall.md

```diff
@@ -24,9 +24,10 @@ function = "increase_balance"
 inputs = [42]
 ```
 
-[//]: # (TODO #1908 is this still valid? Does it refer to old deploy or new deploy <through UDC>?)
 
-Protostar supports two types of transactions — `deploy` and `invoke`, which take similar arguments to [`deploy command`](/docs/cli-reference#deploy) and [`invoke command`](/docs/cli-reference#invoke).
+Protostar supports two types of transactions — `deploy` and `invoke`, which take similar arguments to [`deploy` command](/docs/cli-reference#deploy) and [`invoke` command](/docs/cli-reference#invoke).
+
+When choosing `deploy` transaction type, the deployment happens through the Universal Deployer Contract (UDC), see the [`deploy` command](/docs/cli-reference#deploy) for more information.
```

Comment:
```suggestion
With the `deploy` transaction type, the deployment happens through the Universal Deployer Contract (UDC), see the [`deploy` command](/docs/cli-reference#deploy) for more information.
```

Also we could consider linking to this https://docs.openzeppelin.com/contracts-cairo/0.6.1/udc